security/pkg/nodeagent/plugin/providers/google/stsclient/stsclient.go

```diff
@@ -106,7 +106,7 @@ func (p *SecureTokenServiceExchanger) requestWithRetry(reqBytes []byte) ([]byte,
 			break
 		}
 		monitoring.NumOutgoingRetries.With(monitoring.RequestType.Value(monitoring.TokenExchange)).Increment()
-		stsClientLog.Debugf("token exchange request failed: status code %v, body %v", resp.StatusCode, string(body))
+		stsClientLog.Errorf("token exchange request failed: status code %v, body %v", resp.StatusCode, string(body))
```

Comment:
are we certain the response will never contain a token and leak confidential information into logs?

Reply:
Revised Errorf to print the status code only.